config.py

```diff
@@ -412,18 +412,22 @@ def guess_android_build_tools(sdkdir):
 
 def validate_android_tools(opts):
     if opts['ANDROID_NDK_VERSION'] is None:
-        opts['ANDROID_NDK_VERSION'] = 'r14'
+        opts['ANDROID_NDK_VERSION'] = 'r15'
 
-    ndk_ver = opts['ANDROID_NDK_VERSION']
-    if opts['ANDROID_PLATFORM'] is None:
-        opts['ANDROID_PLATFORM'] = 'android-17'
+    ndk_ver = opts['ANDROID_NDK_VERSION']     
 
     if opts['ANDROID_NDK'] is None:
         ndk = guess_android_tooldir('android-ndk')
         if ndk is None:
             error('Android NDK not found; set $ANDROID_NDK')
         opts['ANDROID_NDK'] = ndk
 
+    if opts['ANDROID_API_VERSION'] is None:
+        opts['ANDROID_API_VERSION'] = '26'
+     
+    if opts['ANDROID_PLATFORM'] is None:
+        opts['ANDROID_PLATFORM'] = 'android-26'   
```

Comment:
@livecodeali would it be a good idea to concatenate `ANDROID_API_VERSION` here so these can't get out of sync?